examples/examples_transitive_deps.bzl

```diff
@@ -4,10 +4,18 @@ There are some transitive dependencies of the dependencies of the examples'
 dependencies. This file contains the required macros to pull these dependencies
```

Comment:
Still says 'extra' :)

Reply:
on Line 1

Reply:
Ooops, I checked `docs_transitive_deps.bzl` instead of examples 😅